Gem/Code/Source/AutoGen/NetworkCoinSpawnerComponent.AutoComponent.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+
+<Component
+    Name="NetworkCoinSpawnerComponent"
+    Namespace="MultiplayerSample"
+    OverrideComponent="true"
+    OverrideController="true"
+    OverrideInclude="Source/Components/NetworkCoinSpawnerComponent.h"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+
+    <ComponentRelation Constraint="Required" HasController="true" Name="NetworkTransformComponent" Namespace="Multiplayer" Include="Multiplayer/Components/NetworkTransformComponent.h" />
+    <ComponentRelation Constraint="Required" HasController="false" HasComponent="true" Name="NetworkPrefabSpawnerComponent" Namespace="MultiplayerSample" Include="Components/PerfTest/NetworkPrefabSpawnerComponent.h" />
+    <ComponentRelation Constraint="Required" HasController="true" HasComponent="true" Name="NetworkRandomComponent" Namespace="MultiplayerSample" Include="Components/NetworkRandomComponent.h" />
+
+    <Include File="AzCore/Asset/AssetSerializer.h"/>
+    <Include File="AzFramework/Spawnable/Spawnable.h"/>
+
+    <ArchetypeProperty Type="AZ::Data::Asset&lt;AzFramework::Spawnable&gt;" Name="CoinSpawnable"
```

Comment:
Consider making a type alias for this inside a core network header that's auto-included. It'll probably happen often enough in customer code, and the xml escape tags are really, really unfortunate..

Reply:
Or in AssetSerializer.h or Spawnable.h for that matter..

Reply:
That is a good idea. Although the discoverability of the special type is still an issue.

Reply:
Folks will generally just copy/paste whatever examples they find in existing components. The more you use that special type in components, the more discoverable it will become.

Reply:
https://github.com/o3de/o3de/pull/11064

Reply:
Done!